packages/content/src/enrichments.ts

```diff
@@ -615,6 +615,7 @@ export const enrichContent = (
         groupIds,
         org,
         server,
+        // TODO: fetch and pass in extent enrichment
```

Comment:
I think I may have found a bug in `enrichContent` since tests started failing once I made my extent changes. The old tests had incorrect assumptions and were passing because of unintended behavior.

Reply:
well, we're trying to kill this, so maybe we won't have to fix it?